include/simulation_lifecycle/structures.h

```diff
@@ -17,4 +17,8 @@ typedef struct viz_options {
     // TODO
 } viz_options_t;
 
+int feature_set_empty(const feature_set_t *p_features);  // TODO
+int relation_set_empty(const relation_set_t *p_relations);  // TODO
+int inconsistent_data(const feature_set_t *p_features, const relation_set_t *p_relations);  // TODO
+
```

Comment:
Not sure I understand what these functions are meant to do. For example : inconsistent_data sounds like it's a function you call when you have inconsistent data so, it will log an error? Then why does it need the two parameters?

Reply:
Ok, I see from main.c, they're checking if the conditions are met by the parameters passed. I'm not sure I'm 100% on board with that pattern...